<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>多点远程超声 - 我的资料</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stacked_widget">
     <widget class="QWidget" name="user_info_page">
      <widget class="QLabel" name="uid_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>71</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>ID：</string>
       </property>
      </widget>
      <widget class="QLabel" name="name_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="name_line_edit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>150</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>561</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="uid_value_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>10</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="hospital_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>71</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>医院：</string>
       </property>
      </widget>
      <widget class="QLabel" name="region_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>45</y>
         <width>71</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>区域：</string>
       </property>
      </widget>
      <widget class="QLabel" name="region_value_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>45</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="hospital_value_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>80</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="phone_line_edit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>200</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="phone_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>手机：</string>
       </property>
      </widget>
      <widget class="QLabel" name="sex_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>性别：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="sex_combo_box">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>69</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="id_line_edit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>300</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="id_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>身份证：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="department_line_edit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>350</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="department_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>科室：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="title_line_edit">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>400</y>
         <width>350</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="title_label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>400</y>
         <width>81</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>职称：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="submit_button">
       <property name="geometry">
        <rect>
         <x>406</x>
         <y>450</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QLabel" name="required_label">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>150</y>
         <width>61</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>（必填）</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="info_page">
      <widget class="QLabel" name="info_label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>170</y>
         <width>561</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="return_button">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>250</y>
         <width>75</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
